index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Visit Uganda Summit</title>
+    <link rel="stylesheet" href="home.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=lato"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="mobile-menu" id="mobile-menu">
+      <div class="mobile-menu-header">
+        <button type="button" class="close-button">&times;</button>
+      </div>
+      <nav>
+        <a href="./index.html" class="navigate"> Home </a>
+        <a href="./about.html" class="navigate"> About </a>
+        <a href="./index.html#program" class="navigate"> Program </a>
+        <a href="#" class="navigate"> Join </a>
+        <a href="./about.html#sponsor" class="navigate"> Sponsor </a>
+      </nav>
+    </div>
```

Comment:
- [OPTIONAL] You could use similar colors to make the mobile menu design more consistent with the overall design.
- [OPTIONAL] Try using a solid background to make the mobile text more readable. In some parts of your menu is kind of hard to read the text.
Your design:
![image](https://user-images.githubusercontent.com/61919617/175123024-c88519bb-ff54-4595-b094-25193f3d530d.png)